react-app/src/Components/HomeScreen/MovieDetails/MovieDetails.jsx

```diff
@@ -72,21 +79,33 @@ function MovieDetails({ openNav, selectedMovie }) {
       </p>
       <div className="flex sm:flex-row flex-col sm:justify-between mt-8 sm:mt-28">
         <div className="flex items-center flex-col sm:flex-row">
-          <div className="sm:ml-0 mx-auto sm:mr-4 sm:mt-0 mt-7 sm:mb-0 mb-3">
-            <AddFavorite fill="#92AAD7" />
-          </div>
-          <div className="text-[#92AAD7] text-lg self-center">Watch Later</div>
-          <Link
-            to={`/trailer/${selectedMovie.id}`}
+          <button
+            type="button"
             className="flex sm:flex-row flex-col items-center"
+            onClick={() => dispatch(addFavoriteMovie(sendRequest, selectedMovie))}
+          >
+            <div className="sm:ml-0 mx-auto sm:mr-4 sm:mt-0 mt-7 sm:mb-0 mb-3">
+              <AddFavorite fill="#92AAD7" />
+            </div>
+            <div className="text-[#92AAD7] text-lg self-center">
+              Watch Later
+            </div>
+          </button>
+          <button
+            type="button"
+            onClick={() => {
```

Comment:
I think it's better to create a function outside the return statement such as `const handleClick = () => {....` for this.